### ISSUE TITLE
Model weights wont converge

### ISSUE THREAD

**Issue by ghanbarzadeh** [hunglc007/tensorflow-yolov4-tflite#379]
I'm trying to train the model on PASCAL 2012, however, the model doesn't seem to converge with my settings (Batch size: 2, Rezolution: 416). This is the highest batch size I could go without my GPU memory running out. 
I was wondering if anybody has had success training with these parameters?

**Comment by ahmethamzakaya** [comment]
Yes i also used small batch on my custom data since my gpu has 4gb vram, but after around 7000 iterations model converged to get above 85% maP  for each class . I recommend to train your model in darknet using " ./darknet detector train command", then convert weights to tensorflow using this repo. 

Also you can refer here for some tricks: https://github.com/AlexeyAB/darknet#how-to-improve-object-detection
